toolkit/bridge/primitives/src/lib.rs

```diff
@@ -40,6 +40,36 @@ pub struct MainChainScripts {
 	pub illiquid_supply_validator_address: MainchainAddress,
 }
 
+#[cfg(feature = "std")]
+impl MainChainScripts {
+	/// Reads the main chain script values from environment
+	///
+	/// It expects the following variables to be set:
+	/// - `BRIDGE_TOKEN_POLICY_ID`
+	/// - `BRIDGE_TOKEN_ASSET_NAME`
+	/// - `ILLIQUID_CIRCULATION_SUPPLY_VALIDATOR_ADDRESS`
+	pub fn read_from_env() -> Result<Self, envy::Error> {
+		#[derive(serde::Serialize, serde::Deserialize)]
+		pub struct MainChainScriptsEnvConfig {
+			pub bridge_token_policy_id: PolicyId,
+			pub bridge_token_asset_name: AssetName,
+			pub illiquid_circulation_supply_validator_address: MainchainAddress,
```

Comment:
The other envy formats used a `String`. Was it because we used to serialize address as hexstring in the past?